bulk-install-app/debug.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+from config import DEBUG
+
+
+def save_debug_info(folder, name, response):
+    if DEBUG:
+        if not os.path.exists(f"{folder}"):
+            os.makedirs(f"{folder}")
+
+        file = open(f"{folder}/{name}-content.html", "w")
```

Comment:
Can we use `with open` so we don't need to worry about closing the file when there is a problem writing content?

Reply:
Done!